### PR TITLE
Add own projects to project section

### PR DESCRIPTION
**Pull request by DavidMatalik**
And also adjust some styling issues.
Peek 2021-06-03 18-39

### CODE REVIEW

**Comment by DavidMatalik** [comment]
Questions: 
1. You could see slider arrow rather badly. Ok as it is now in black? Or better colouring ideas?

2. Because of white Weather image background: added border below project images in general. Ok like this?

File: public/res_primaryLanguage.json [hunk 1]
```diff
@@ -11,26 +11,22 @@
     },
     "projects": [
       {
-        "title": "Animal Shelter",
-        "startDate": "2020",
-        "description": "The most expanded application I had opportunity to work with. I've learned many technologies and my code was reviewed by awesome curator. Application handles all adoption processess and allows to store all evidence on adopting animals from animal shelter.",
+        "title": "Todo App",
+        "startDate": "2021",
+        "description": "I learned a lot about SOLID principles doing this Odin bootcamp project. I love to be organized. That is why with a friend developer I build an extended todo app - with Firebase background structure, and more functionality. We also created an UML diagramm for optimizing the architecture.",
```
Comment (RichStone) [inline public/res_primaryLanguage.json]: "I love to be organized (GTD rocks!)."

Maybe you can already add the link to the repo of the new app even if it's not ready yet

File: src/components/ProjectDetailsModal.js [hunk 2]
```diff
@@ -14,19 +17,35 @@ class ProjectDetailsModal extends Component {
...
-                    <p className="text-center" style={{ fontSize: "30%" }}>
-                      {icons.name}
-                    </p>
-                  </i>
-                </div>
-              </span>
-            </li>
-          );
+          // For using icon from iconify
+          if (icons.name === "api"){
```
Comment (RichStone) [inline src/components/ProjectDetailsModal.js:21]: smells of duplication :) I think, in future we should consider making these their own components

File: src/App.scss [hunk 3]
```diff
@@ -157,6 +157,12 @@ header h1 {
 	top: 15px;
 }
 
+// Adjust color of slider arrows
+.awssld__controls{
+  --organic-arrow-color: black !important;
```
Comment (RichStone) [inline src/App.scss:162]: I somehow like the black contrast here, but probably you could see if you like a light grey with a thin black border more (or without border but with a tiny shadow)

**Review by RichStone — APPROVED** [review]

**Comment by RichStone** [comment]
> 2. Because of white Weather image background: added border below project images in general. Ok like this?

I think it's a nice tweak!

good texts and seems cool to me in general! :+1: (thanks for the gifs :))

File: src/App.scss (hunk 3 above)
Comment (DavidMatalik) [reply src/App.scss:162]: I tried a bit, but I can't figure out how to access border-property... this comes from awesome slider module. And in dev tools nothing happens when adding border property or --organic-arrow-border property.

File: public/res_primaryLanguage.json (hunk 1 above)
Comment (DavidMatalik) [reply public/res_primaryLanguage.json]: Nice addition with GTD =). 

We really haven't come far yet with implementing stuff... Maybe at a later point I would add a link...

File: src/components/ProjectDetailsModal.js (hunk 2 above)
Comment (DavidMatalik) [reply src/components/ProjectDetailsModal.js:21]: You mean when the next case like this is needed?

**Pull request merged**